src/main/java/com/amazon/opendistroforelasticsearch/security/auth/BackendRegistry.java

```diff
@@ -207,8 +206,8 @@ public void invalidateCache() {
         transportImpersonationCache.invalidateAll();
     }
 
-    @Override
-    public void onChanged(ConfigModel cm, DynamicConfigModel dcm, InternalUsersModel ium) {
+    @Subscribe
+    public void onDynamicConfigModelChanged(DynamicConfigModel dcm) {
```

Comment:
How does this know which event bus to subscribe to? 
What happens if I make another event bus and post DynamicConfigModel ? Does this automatically subscribe for both?

Reply:
It is necessary to register with the event bus. Please see `DynamicConfigFactory.registerDCFListener()`.